cablab-singleuser-julia-nb/Dockerfile

```diff
@@ -28,3 +28,4 @@ RUN /srv/jupyterhub/julia/bin/julia precomp.jl
 
 USER $NB_USER
 WORKDIR /home/jovyan/work
+ENV ESDL_WORKDIR /home/jovyan/work
```

Comment:
ESDL_WORKDIR  has been assigned a few lines before to `/home/jovyan/tmp`. If this is only for some intermediate files, I would suggest to use `tmp` rather than `work`. And I think using `ENV` is a better way than `RUN export ...`.

Reply:
Sorry, my bad I did not look close enough. Yes, probably tmp is a better directory.